api/src/main/java/org/pmiops/workbench/db/dao/CohortService.java

```diff
@@ -0,0 +1,31 @@
+package org.pmiops.workbench.db.dao;
+
+import org.pmiops.workbench.db.model.Cohort;
+import org.pmiops.workbench.db.model.CohortReview;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+
+public class CohortService {
+  // Note: Cannot use an @Autowired constructor with this version of Spring
+  // Boot due to https://jira.spring.io/browse/SPR-15600. See RW-256.
+  @Autowired private CohortDao cohortDao;
+  @Autowired private CohortReviewDao cohortReviewDao;
+
+  @Transactional
+  public Cohort saveAndCloneReviews(Cohort from, Cohort to) {
+    Cohort saved = cohortDao.save(to);
+    for (CohortReview fromReview : from.getCohortReviews()) {
+      CohortReview cr = new CohortReview();
```

Comment:
We also need to clone ParticipantCohortStatus rows under CohortReview. Note that there can be up to 10K of these per review, and @freemabd1 has run into an issue where batch inserting these into Cloud SQL is slow (he is investigating the cause now.)

Reply:
Added via native sql, see `ParticipantCohortStatusDao`.